engine/src/main/java/org/terasology/rendering/dag/tasks/SetFacesToCullTask.java

```diff
@@ -22,10 +22,10 @@
 /**
  * TODO: Add javadocs
  */
-public final class SetWhatFacesToCullTask implements RenderPipelineTask {
+public final class SetFacesToCullTask implements RenderPipelineTask {
```

Comment:
Doh. Of course they can't be private otherwise they can't get into the tasklist.